<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExQTreeWidget</class>
 <widget class="QMainWindow" name="ExQTreeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>601</width>
         <height>489</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actAddFolder"/>
   <addaction name="actAddFile"/>
   <addaction name="actDelFile"/>
   <addaction name="actThroughFile"/>
   <addaction name="actEnlarge"/>
   <addaction name="actShrink"/>
   <addaction name="actFitHeight"/>
   <addaction name="actFitWidth"/>
   <addaction name="actFloat"/>
   <addaction name="actHide"/>
   <addaction name="actExit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>271</width>
       <height>500</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">节点</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>节点类型</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <action name="actAddFolder">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/Image016.jpg</normaloff>:/image/Image016.jpg</iconset>
   </property>
   <property name="text">
    <string>添加文件夹</string>
   </property>
   <property name="toolTip">
    <string>添加文件夹</string>
   </property>
  </action>
  <action name="actAddFile">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/Image007.jpg</normaloff>:/image/Image007.jpg</iconset>
   </property>
   <property name="text">
    <string>添加文件</string>
   </property>
   <property name="toolTip">
    <string>添加文件</string>
   </property>
  </action>
  <action name="actDelFile">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/Image005.jpg</normaloff>:/image/Image005.jpg</iconset>
   </property>
   <property name="text">
    <string>删除文件</string>
   </property>
   <property name="toolTip">
    <string>删除文件</string>
   </property>
  </action>
  <action name="actThroughFile">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/Image008.jpg</normaloff>:/image/Image008.jpg</iconset>
   </property>
   <property name="text">
    <string>遍历节点</string>
   </property>
   <property name="toolTip">
    <string>遍历节点</string>
   </property>
  </action>
  <action name="actEnlarge">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/Image012.jpg</normaloff>:/image/Image012.jpg</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
  </action>
  <action name="actShrink">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/Image011.jpg</normaloff>:/image/Image011.jpg</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="actFitHeight">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/Image009.jpg</normaloff>:/image/Image009.jpg</iconset>
   </property>
   <property name="text">
    <string>最佳高度</string>
   </property>
  </action>
  <action name="actFitWidth">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/Image010.jpg</normaloff>:/image/Image010.jpg</iconset>
   </property>
   <property name="text">
    <string>最佳宽度</string>
   </property>
   <property name="toolTip">
    <string>最佳宽度</string>
   </property>
  </action>
  <action name="actFloat">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/Image015.jpg</normaloff>:/image/Image015.jpg</iconset>
   </property>
   <property name="text">
    <string>窗口浮动</string>
   </property>
   <property name="toolTip">
    <string>窗口浮动</string>
   </property>
  </action>
  <action name="actHide">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/Image003.jpg</normaloff>:/image/Image003.jpg</iconset>
   </property>
   <property name="text">
    <string>隐藏</string>
   </property>
   <property name="toolTip">
    <string>隐藏</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/Image004.jpg</normaloff>:/image/Image004.jpg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
